autoload/lsp/ui/vim/signature_help.vim

```diff
@@ -38,8 +38,32 @@ function! s:handle_signature_help(server, data) abort
     endif
 
     if !empty(a:data['response']['result']) && !empty(a:data['response']['result']['signatures'])
-        let l:signature = a:data['response']['result']['signatures'][0]
-        let l:contents = [l:signature['label']]
+        " Get current signature.
+        let l:signatures = get(a:data['response']['result'], 'signatures', [])
+        let l:signature_index = get(a:data['response']['result'], 'activeSignature', 0)
+        let l:signature = get(l:signatures, l:signature_index, {})
+        if empty(l:signature)
+            return
+        endif
+
+        " Signature label.
+        let l:label = l:signature['label']
+
+        " Mark current parameter.
+        if has_key(a:data['response']['result'], 'activeParameter')
+            let l:parameters = get(l:signature, 'parameters', [])
+            let l:parameter_index = a:data['response']['result']['activeParameter']
+            let l:parameter = get(l:parameters, l:parameter_index, {})
+            let l:parameter_label = get(l:parameter, 'label', '')
```

Comment:
This can also be `[number, number]`, so we should handle that case as well.

Reply:
Thanks for review.
Sorry, I missed `[number, number]`.

I implemented, But I don't know the server that return `[number, number]`.
Could you know the server?

Anyway, I will update this PR.

Reply:
I tested a couple, haven't found one yet, unfortunately. For now, you can test it with a hardcoded value, I guess.